diffsims/generators/sphere_mesh_generators.py

```diff
@@ -481,5 +481,5 @@ def beam_directions_grid_to_euler(vectors):
     phi2 = sign * np.nan_to_num(np.arccos(x_comp / norm_proj))
     # phi1 is just 0, rotation around z''
     phi1 = np.zeros(phi2.shape[0])
-    grid = np.rad2deg(np.vstack([phi1, Phi, phi2]).T)
+    grid = np.rad2deg(np.vstack([phi1, Phi, np.pi/2 - phi2]).T)
```

Comment:
This was the bugfix? Could you remind me what's going on?

Reply:
Exactly. This is to match the triangle of beam directions to the ones defined. I found there was a 90 degree offset in the original implementation. Without the fix, the triangle for cubic looks like:
![image](https://user-images.githubusercontent.com/25320842/99443615-5a410b00-291b-11eb-9c47-0151f0c6d5a1.png)
This is the triangle 001 - 111 - 011.

With the fix it looks like:
![image](https://user-images.githubusercontent.com/25320842/99442873-45b04300-291a-11eb-8aae-66c6a189f05f.png)
So the triangle is between (001) - (111) - (101) as defined in `rotation_list_generators`

I checked also for the other crystal systems whether it's correct

Reply:
Clear improvement, good to see.